### ISSUE TITLE
Can't assign a winner or copy edit options for ballot measures

### ISSUE THREAD

**Issue by wmandrews** [newsdev/elex-admin#40]
http://int-elex-prd-east.newsdev.net/elections/2016/admin/2016-06-07/race/CA-8692/

**Comment by jeremyjbowers** [comment]
Yeah, for non-national races you can do basically nothing. The solution to this is my general elections problem as well.
